tests/Integration/Database/EloquentUpdateTest.php

```diff
@@ -112,7 +112,7 @@ public function testIncrement()
             'counter' => 0,
         ])->delete();
 
-        TestUpdateModel3::increment('counter');
+        TestUpdateModel3::query()->increment('counter');
```

Comment:
The previous behaviour is no longer allowed, similar to `TestUpdateModel3::update([...])` not being possible either.